core/client/app/styles/components/selectize.css

```diff
@@ -4,13 +4,16 @@
     background: #f2f2f2 !important;
     background: rgba(0, 0, 0, 0.06) !important;
 }
-.selectize-control.plugin-drag_drop .ui-sortable-placeholder::after {
```

Comment:
This surprised me so I had to look it up. It seems the single-colon for pseudo elements is a hang-over from CSS 1/2, with CSS 3 using the double-colon to differentiate pseudo-elements (`::`) from pseudo-classes (`:`).

If we are standardising on the single-colon selector (which user agents need to adhere to for compatibility reasons) it's necessary to remember that any pseudo elements introduced in CSS 3 or later (e.g., `::selection` and `::backdrop`)  must always use the double-colon.

Ref: http://www.w3.org/TR/selectors/#pseudo-elements